app/models/friend_request.rb

```diff
@@ -0,0 +1,29 @@
+class FriendRequest < ApplicationRecord
+  scope :pending, -> { where(status: 0) }
+
+  validates :sender, presence: true
```

Comment:
We don't need to validate the presence of `sender` and `receiver` as the `belongs_to` associations are doing that for us.